src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardSamlBundle/Security/Authentication/Provider/SamlProvider.php

```diff
@@ -93,7 +93,12 @@ public function authenticate(TokenInterface $token)
 
         $nameId = $translatedAssertion->getNameID();
         $email = $this->getSingleStringValue('mail', $translatedAssertion);
-        $commonName = $this->getSingleStringValue('commonName', $translatedAssertion);
+
+        try {
+            $commonName = $this->getSingleStringValue('commonName', $translatedAssertion);
+        } catch (BadCredentialsException $e) {
+            $commonName = '';
```

Comment:
Please add an info log entry for this situation. We are no longer logging the attribute not found/bad credentials error in the logs. But this situation does deserve a log mention in my opinion.

Reply:
Added logging for this fallback mechanism.

Reply:
Nice upgrade!